.travis.yml

```diff
@@ -1,5 +1,10 @@
+sudo: false
+
 language: node_js
 
+node_js:
+  - "node"
```

Comment:
Without that the build wasn't able to step forward from `nvm install 0.10`.
See Travis CI [build #173][1]!

[1]: https://travis-ci.org/mozilla/memchaser/builds/89566396